docs/pages/reference/signature-algorithms.mdx

```diff
@@ -215,3 +215,56 @@ The signature algorithm suites are designed to simplify the configuration burden
 We did not want to expose 100 configuration knobs to modify every single
 signature Teleport does, which could lead to thousands of possible combinations
 we'd have to support, and could create the possibility for insecure combinations.
+
+### What if my HSM doesn't support ECDSA keys?
+
+If you are using a PKCS#11 HSM that does not support ECDSA keys, we recommend
+you continue to use the `legacy` algorithm suite.
+For an existing cluster it's possible you could switch to the `hsm-v1` suite and
+users and hosts could begin to use new key algorithms while your CAs continue to
+use their existing RSA keys, but whenever a new CA key is generated it may fail
+if the CA uses non-RSA keys.
```

Comment:
As an HSM beginner I'm not sure to understand this sentence.

Does this mean I can:
- seamlessly transition from an old RSA-only CA to a new CA supporting RSA and Elliptic
- keep using my old RSA-only CA with the `hsm-v1` suite
- use the `hsm-v1` suit with my old RSA-only CA but things will fail under specific circumstances (which ones?)

Reply:
reworded this to

> If you have a strong need to have users and hosts use Ed25519 and/or ECDSA keys,
> you could switch to the `hsm-v1` suite, but whenever a new CA key is generated
> it may fail if the CA uses non-RSA keys.
> This may cause the Auth Service to terminate with an error after trying to
> generate the key.
> New CA keys may be generated the first time a new Auth Service starts, when a CA
> rotation is initiated, or during a version upgrade when a new CA has been added
> in the new version.
> For these reasons we recommend continuing to use the `legacy` algorithm suite.